personio-personnel-data-api-oa3.yaml

```diff
@@ -3983,3 +4105,722 @@ components:
         - link
         - tags
         - multiline
+
+    PublicListReportsResponse:
```

Comment:
[Question]: This response contains information about pagination. The endpoint definitions you are providing are not describing any `limit` or `offset` fields. Are you going to support pagination for these endpoints? If so, please provide definitions for the `limit` and `offset` fields for all endpoints where you want to allow pagination.

Reply:
We do not support pagination now. That will be added in the next version, but these fields are being used for the client to know the number of results of the request.